rdpy/core/type.py

```diff
@@ -795,7 +796,11 @@ def __write__(self, s):
             toWrite += self._until
             
         if self._unicode:
-            s.write(encodeUnicode(self.value))
+            val = self.value
```

Comment:
Why alias this value here? It's already been aliased above and then mutated, so shouldn't we be using that?